xml/System/Span`1.xml

```diff
@@ -403,9 +461,14 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns a reference to the element of the <see cref="T:System.Span`1" /> at index zero.</summary>
```

Comment:
The return type needs to be `ref T`, not T. Similary, span is a `ref struct`, not a regular struct.

@rpetrusha, do we generally make a distinction within the docs when a method is marked with a `EditorBrowsableState.Never` attribute?

@jkotas, should we call out the fact that this method is unsafe within the docs?

Reply:
The doc should say that this method is there to support C# `fixed` statement.

There is nothing unsafe about this method (calling this method directly won't let you violate type or memory safety). I do not think the doc for this method needs to say anything about safety.

We have moved all unsafe Span-related methods to `System.Runtime.InteropServices` namespace. The documentation for those should mention that they are unsafe.

Reply:
/cc @dend Do you pull the `ref` from the return type, or make a distinction between `struct` and `ref struct` when processing the ref assemblies?